frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
can we please update the readme with some screenshots of the app, what it does and some comments on how to use it locally?